class.json-api-endpoints.php

```diff
@@ -1421,13 +1426,19 @@ function load_theme_functions() {
 		if ( defined( 'REST_API_THEME_FUNCTIONS_LOADED' ) )
 			return;
 
+		// VIP context loading is handled elsewhere, so bail to prevent
+		// duplicate loading. See `switch_to_blog_and_validate_user()`
+		if ( function_exists( 'wpcom_is_vip' ) && wpcom_is_vip() ) {
+			return;
+		}
+
 		define( 'REST_API_THEME_FUNCTIONS_LOADED', true );
 
 		// the theme info we care about is found either within functions.php or one of the jetpack files.
 		$function_files = array( '/functions.php', '/inc/jetpack.compat.php', '/inc/jetpack.php', '/includes/jetpack.compat.php' );
 
 		$copy_dirs = array( get_template_directory() );
-		if ( wpcom_is_vip() ) {
+		if ( function_exists( 'wpcom_is_vip' ) && wpcom_is_vip() ) {
```

Comment:
@dereksmart This is kinda strange, don't you think? This will never be true because of the `return` a couple lines up.